core/src/toga/window.py

```diff
@@ -950,6 +942,8 @@ def closeable(self) -> bool:
 
 
 class MainWindow(Window):
+    """Create a new main window."""
+
```

Comment:
This causes a duplicate paragraph in the docs.

Reply:
Dangit... stray leftovers from my docstring experiments